addons/vehicles/CfgEventHandlers.hpp

```diff
@@ -32,4 +45,9 @@ class Extended_Init_EventHandlers {
             exclude[] = {"TargetSoldierBase","Static_Designator_01_base_F","Static_Designator_02_base_F","Pod_Heli_Transport_04_base_F"};
         };
     };
+    class AllVehicles {
+        class ACE_EngineState {
+            init = QUOTE(if (local (_this select 0)) then {(_this select 0) setVariable [ARR_3(QQGVAR(engineState),isEngineOn (_this select 0),true)]});
```

Comment:
Same thing as above with different value. Function!

Reply:
Do we even need this? I am pretty sure the engine is always off on a created vehicle.

Reply:
This is an Init event. The script uses `setVariable public`. `setVariable public` does not work on the Init event: the variable will not be synchronized in multiplayer.

Reply:
Is it always off? Think of vehicles placed in the air. They don't have any spin-up but are at full power from the start. I'm also not quite sure and just added this as we always start our missions from the ground. But I can test is.

Reply:
In any case, setVariable public does not work from Init.

Reply:
I got that and will fix that. Probably just move the locality check to the config.